arches/app/models/graph.py

```diff
@@ -373,7 +375,13 @@ def save(self, validate=True, nodeid=None):
                 node = self.nodes[nodeid]
                 self.update_es_node_mapping(node, datatype_factory, se)
                 self.create_node_alias(node)
-                node.save()
+                try:
+                    node.save()
+                except IntegrityError as err:
+                    if "unique_alias_graph" in str(err):
+                        message = _('Duplicate node alias: "{0}". All aliases must be unique in a resource model.'.format(node.alias))
+                        raise GraphValidationError(message)
+
```

Comment:
We should probably add a logger.error(err) if error is not a unique_alias_graph.